pyairtable/api/abstract.py

```diff
@@ -9,22 +9,25 @@
 
 from .params import to_params_dict
 
+DEFAULT_MAX_RECORDS_PER_REQUEST = 10
+
 
 class ApiAbstract(metaclass=abc.ABCMeta):
     VERSION = "v0"
     API_BASE_URL = "https://api.airtable.com/"
     API_LIMIT = 1.0 / 5  # 5 per second
     API_URL = posixpath.join(API_BASE_URL, VERSION)
-    MAX_RECORDS_PER_REQUEST = 10
 
     session: requests.Session
-    tiemout: Optional[Tuple[int, int]]
+    timeout: Optional[Tuple[int, int]]
 
-    def __init__(self, api_key: str, timeout=None):
+    def __init__(self, api_key: str, timeout: Optional[int]=None,
+                 max_records_per_request: Optional[int]=DEFAULT_MAX_RECORDS_PER_REQUEST):
```

Comment:
This isn't black formatted. You ought to install the packages in `requirements-dev.txt` and then run 
```sh
black .
````

Reply:
Related:
https://github.com/gtalarico/pyairtable/pull/144#discussion_r826602036

Reply:
created a fresh CI to test CI, and seem to catch formatting, not sure why this CI steps not running on this PR:
https://github.com/gtalarico/pyairtable/pull/151

Reply:
@gtalarico The branch needs to be rebased on master to have the changes to the .github folder picked up.